science/imager/files/patch-admin-gildas-env.sh.diff

```diff
@@ -0,0 +1,95 @@
+--- gildas-src-dec22a/admin/gildas-env.sh.orig	2022-12-01 05:00:13
++++ gildas-src-dec22a/admin/gildas-env.sh	2022-12-02 09:35:21
+@@ -238,19 +238,19 @@
+ 	FFTW3_PRESENT=no
+ 	FFTW3F_PRESENT=no
+ 	#
+-	OLDIFS=$IFS
+-	IFS=:
+-	for DIR in $GAGENV_SEARCH_PATH; do
+-	    # Note: libfftw3 and libfftw3f must be in the same directory
+-	    if file_present "libfftw3." "${DIR}"; then
+-		FFTW3_PRESENT=yes
+-		FFTW3_LIB_DIR=$DIR
+-		if file_present "libfftw3f." "${DIR}"; then
+-		    FFTW3F_PRESENT=yes
+-		fi
+-	    fi
+-	done
+-	IFS=$OLDIFS
++# 	OLDIFS=$IFS
++# 	IFS=:
++# 	for DIR in $GAGENV_SEARCH_PATH; do
++# 	    # Note: libfftw3 and libfftw3f must be in the same directory
++# 	    if file_present "libfftw3." "${DIR}"; then
++# 		FFTW3_PRESENT=yes
++# 		FFTW3_LIB_DIR=$DIR
++# 		if file_present "libfftw3f." "${DIR}"; then
++# 		    FFTW3F_PRESENT=yes
++# 		fi
++# 	    fi
++# 	done
++# 	IFS=$OLDIFS
```

Comment:
Why write a patch that comments out lines? It's simpler, shorter, and easier to read to write a patch that deletes the lines.

Reply:
we intent to support FFTW3 even in openMP mode in the future so we keep this code as a comment for reference